### PR TITLE
chore: ci/cd help messaging

### PR DESCRIPTION
**Pull request by hugorut**
- [x]  additional data processing clarification
- [x]  Expand CI/CD detection
- [x]  Mention CI/CD in CLI runs
- [x]  Update Infracost --help to include link to CI/CD

### CODE REVIEW

**Comment by hugorut** [comment]
ah interesting the PR is failing as it's detecting the CI system and not appending the help comment. I'll fix to detect if in test as well

File: cmd/infracost/run.go [hunk 1]
```diff
@@ -326,7 +327,7 @@ func runProjectConfig(cmd *cobra.Command, runCtx *config.RunContext, ctx *config
 		NoColor:       runCtx.Config.NoColor,
 		Indent:        "  ",
 	}
-	spinner := ui.NewSpinner("Calculating monthly cost estimate", spinnerOpts)
+	spinner := ui.NewSpinner("Extracting only cost-related params from terraform plan\nRetrieving cloud prices to calculate costs", spinnerOpts)
```
Comment (aliscott) [inline cmd/infracost/run.go:330]: Should this be two separate spinners? The first one (`Extracting only cost-related params...`) inside the `LoadResources` call starting after the plan JSON has been generated, and the second one (`Retrieving cloud prices`) here.

Comment (hugorut) [reply cmd/infracost/run.go:330]: hmm in the implementation docs it's as such:

```
  ✔ Extracting only cost-related params from terraform plan
     Retrieving cloud prices to calculate costs
 ```
 
 paging dr @alikhajeh1 - thoughts?

Comment (aliscott) [reply cmd/infracost/run.go:330]: Ah okay, I left a comment on the original docs that suggested this option as two separate spinners, but it looks that comment was resolved so wasn't appearing any more.

I don't think having multi-lines works with our current spinner library anyway.

Comment (hugorut) [reply cmd/infracost/run.go:330]: 👍  - I'll alter

**Comment by aliscott** [comment]
Screenshot 2022-01-30 at 15 45 22

@alikhajeh1 @hugorut seems like a lot of links at the bottom now. I'm wondering if we want a different color for primary and secondary links, but maybe we do this when we fix https://github.com/infracost/infracost/issues/1175.

**Review by aliscott — APPROVED** [review]
LGTM pending the spinner changes!

**Comment by alikhajeh1** [comment]
@hugorut can you please review my latest commits? If that wording reads ok for you, can you possibly update the tests and merge this?

@aliscott / @hugorut 4 links colored the same does indeed seem like a lot (usage-file, new resource issue, share link, CI/CD). I made the top 2 links white so they don't stand out just now as those are not the primary actions. How's that?
Screen Shot 2022-01-30 at 10 58 12 AM

**Comment by hugorut** [comment]
> @aliscott / @hugorut 4 links colored the same does indeed seem like a lot (usage-file, new resource issue, share link, CI/CD). I made the top 2 links white so they don't stand out just now as those are not the primary actions. How's that?

Looks good 👍

**Comment by alikhajeh1** [comment]
@hugorut infracost.io/cicd is live so this can be shipped, I'll change the URL to goto a new dedicated CI page Mon/Tue

**Comment by hugorut** [comment]
Merging this - the failing test is to do with the frontdoor issue, we're following here: https://github.com/hashicorp/terraform-provider-azurerm/issues/15170

**Pull request merged**